python/ray/train/examples/tune_cifar_torch_pbt_example.py

```diff
@@ -13,11 +13,11 @@
 import ray.train as train
 from ray import tune
 from ray.air.config import FailureConfig, RunConfig, ScalingConfig
+from ray.air.tests.resnet import ResNet18
```

Comment:
nit: prefer not to import from tests (keep tests as a non-module)

Reply:
fyi, the torchvision resnet performs about 20% worse than our custom one.. just fyi